docs/chapters/baseplatform/replacing-packagegroups.rst

```diff
@@ -0,0 +1,63 @@
+:orphan:
+
+.. _replacing-packagegroups:
+
+Replacing packagegroups by hand-picked components
+==================================================
+
+By default PELUX include components from Qt Auto using the packagegroups feature.
+This chapter describes how to replace these default Qt Auto components with
+hand-picked components in a new distribution.
+
+The following packagegroups from Qt Auto are included in the PELUX image.
+
+* ``packagegroup-b2qt-automotive-addons``
+* ``nativesdk-packagegroup-b2qt-automotive-qt5-toolchain-host``
+* ``packagegroup-b2qt-automotive-qt5-toolchain-target``
+ 
+These packagegroups add several Qt Auto add-ons (includes ``neptune-ui`` among others) 
+and packages that are used when building SDK for host and target machines.
+
+If all of the Qt Auto components defined in above packagegroups are needed in the new 
+distribution then image recipe ``meta-<distro>/<image-recipe-path>/<image-name>.bb`` of 
+the new distribution can simply include ``core-image-pelux-qtauto-neptune`` as follows.
+
+.. code-block:: bash
+
+    DESCRIPTION = "New Distro image with QtAuto frontend"
+
+    require core-image-pelux-qtauto-neptune
```

Comment:
I believe it is usually recommended to use a "full" path here as `recipes-core/images/core-image-pelux-qtauto-neptune.bb`